src/applications/mhv/landing-page/api/mocks/user/index.js

```diff
@@ -124,17 +124,16 @@ const cernerUser = {
   meta: { errors: null },
 };
 
-const generateUserWithFacilities = ({ facilities = [], name = 'Harry' }) => {
+const generateUserWithFacilities = ({ facilities = [] }) => {
   return {
     ...defaultUser,
     data: {
       ...defaultUser.data,
       attributes: {
         ...defaultUser.data.attributes,
-        profile: {
-          ...defaultUser.data.attributes.profile,
+        va_profile: {
```

Comment:
Platform's `mapRawUserDataToState` & redux handles translating `va_profile` fields for `store.user.profile`. We should match the API response with our mocks.